### PR TITLE
java_cert: support of different keystore type (JKS, JCEKS)

### PR DESCRIPTION
**Pull request by kharkevich**
##### SUMMARY
Add: Support of different keystore type (JKS, JCEKS)

##### ISSUE TYPE

- Docs Pull Request
- Feature Pull Request

##### COMPONENT NAME

java_cert

##### ADDITIONAL INFORMATION

Example of using provided in documentation section

```paste below

```

### CODE REVIEW

File: lib/ansible/modules/system/java_cert.py [hunk 1]
```diff
@@ -132,19 +149,26 @@
 from ansible.module_utils.basic import AnsibleModule
 
 
-def check_cert_present(module, executable, keystore_path, keystore_pass, alias):
+def get_keystore_type(keystore_type):
+    ''' Check that custom keystore is presented in parameters '''
+    if not keystore_type:
```
Comment (barlik) [inline lib/ansible/modules/system/java_cert.py]: Is this correct? this would only be true when keystore_type is empty.

Comment (kharkevich) [reply lib/ansible/modules/system/java_cert.py]: My bad, wrong rework from lambda. Will be fixed now

**Comment by kharkevich** [comment]
CC @jborean93

**Review by arno49 — APPROVED** [review]

**Comment by kharkevich** [comment]
cc @bcoca

**Comment by gundalow** [comment]
@kharkevich Thank you for this PR
@arno49 Thank you for the review

rebuild_merge
This should be merged shortly by the Bot for release in  Ansible 2.8

**Pull request merged**